devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy

```diff
@@ -84,6 +84,9 @@ def configure() {
 		// Tell sensor to send us battery information instead of USB power information
 		commands << encap(zwave.configurationV1.configurationSet(parameterNumber: 0x5E, scaledConfigurationValue: 1, size: 1))
 		response(delayBetween(commands, 1000) + ["delay 20000", encap(zwave.wakeUpV1.wakeUpNoMoreInformation())])
+	} else if (zwaveInfo.mfr.equals("021F") || zwaveInfo.mfr.equals("0258")) {
```

Comment:
Same as above, let's create `is<mfr name>` or `is<mfr name><model name>` functions for these.

Reply:
Done